### PR TITLE
Small improvements to the eventing-controller dev setup

### PR DESCRIPTION
**Pull request by pxsalehi**
**Description**

Small improvements to the eventing-controller dev setup

Changes proposed in this pull request:

- Use correct make target name in the readme file
- Use the same name/label for the ko-based eventing-controller. This seems to make it easier to deploy the ko-based controller on an existing cluster (no need to remove the existing deployment first).

**Related issue(s)**

### CODE REVIEW

File: components/eventing-controller/README.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh |
 > Existing deployment for eventing-controller should be deleted from the Kyma cluster which can interfere with reconciliation process.
 
 ```sh
-make DOMAIN=custom-domain.com deploy-local
+make DOMAIN=custom-domain.com deploy
 ```
 
 - To verify all the manifests after the processing by Kustomize without applying to the cluster, use make target `deploy-local-dry-run`:
```
Comment (sayanh) [inline components/eventing-controller/README.md:30]: Let's change the target here as well.
```suggestion
- To verify all the manifests after the processing by Kustomize without applying to the cluster, use make target `deploy-dry-run`:
```

**Review by sayanh — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**